packages/css/test/errors.ts

```diff
@@ -0,0 +1,52 @@
+import { expecter } from 'ts-snippet'
```

Comment:
I'm feeling kinda bad installing a library with 16 stars, but no other solution I'm aware of (e.g. tsd, dtslint, dts-jest) integrates with existing Jest setup with no additional config like ts-snippet does.

Reply:
Eh, the author looks to be very active on GitHub - probably safe (and certainly convenient).